extensions/amp-bind/amp-bind.md

```diff
@@ -204,6 +204,7 @@ Only binding to the following components and attributes are allowed. Most bindab
 | `<amp-iframe>` | `[src]` | Changes the iframe's source URL. |
 | `<amp-img>` | `[alt]`<br>`[attribution]`<br>`[src]`<br>`[srcset]` | See corresponding [amp-img attributes](https://www.ampproject.org/docs/reference/components/media/amp-img#attributes). |
 | `<amp-selector>` | `[selected]`<sup>1</sup> | Changes the currently selected children element(s)<br>identified by their `option` attribute values. Supports a comma-separated list of values for multiple selection. [See an example](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind).
+| `<amp-state>` | `[src]` | Fetches JSON from the new URL and merges it into the existing state. |
```

Comment:
If we're going to allow binding to this attribute on `<amp-state>` elements that don't have the `src` attribute, we might want to spell that out explicitly here.